presto-docs/src/main/sphinx/release/release-0.211.rst

```diff
@@ -0,0 +1,51 @@
+=============
+Release 0.211
+=============
+
+General Changes
+---------------
+
+* Fix missing final query plan in ``QueryCompletedEvent``. Statistics and cost estimates
+  are removed from the plan text because they may not be available during event generation.
+* Update the default value of the ``http-server.https.excluded-cipher`` config
+  property to exclude non FS cipher suites. Specifically, all
+  ciphers without forward secrecy, and all ciphers that use the RSA key exchange
+  algorithm are excluded by default. Consequently, TLS 1.0 or TLS 1.1 are no
+  longer supported by default. The ``http-server.https.excluded-cipher`` config property
+  can be set to empty string to restore the old behavior.
+* Add :func:`ST_GeomFromBinary` and :func:`ST_AsBinary` functions that convert
+  geometries to and from Well-Known Binary format.
+* Remove the ``verbose_stats`` session property, and rename the ``task.verbose-stats``
+  configuration property to ``task.per-operator-cpu-timer-enabled``.
+* Remove the ``experimental.enable-new-stats-calculator`` configuration property.
```

Comment:
Might be worth mentioning that a related `experimental.enable-stats-calculator` switch has been added. @arhimondr

Reply:
Actually, why do we even document in release notes these changes to experimental configuration properties. We didn't document its addition either. I'll remove this line.